src/linear-progress.jsx

```diff
@@ -146,6 +163,7 @@ const LinearProgress = React.createClass({
         borderRadius: 2,
         margin: 0,
         overflow: 'hidden',
+        visibility: this.state.visible ? 'visible' : 'hidden',
```

Comment:
I chose to use CSS visibility, so that the progress indicator takes up a visual space even when it's not there. If I render nothing, or use `display: none`, there's an unattractive "jump" as the space expands to show the progress indicator.